app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/Router.scala

```diff
@@ -167,6 +167,14 @@ trait JourneyTypeRoutes extends Product with Serializable {
   def nextPageForCheckBankAccountDetails(): Call =
     claimRoutes.SelectBankAccountTypeController.selectBankAccountType(journeyBindable)
 
+  def nextPageForEnterBankAccountDetails(
+    isBankAccountType: Boolean
```

Comment:
I don't like this name.
What does it even mean?
Did you want something like isBusinessAccount? or isChange?

Reply:
The isBankAccountType is for knowing whether the user has entered a bank account type or not .  Maybe hasBankAccountType?

Reply:
Yeah, `hasBankAccountType` sounds better